specification/web/resource-manager/Microsoft.Web/stable/2018-11-01/WebApps.json

```diff
@@ -4585,6 +4585,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/listbackups": {
+      "post": {
```

Comment:
adding new apis in existing stable versions is breaking change, suggest to add in new release

Reply:
Why is adding a new API a breaking change? Existing APIs are untouched.

Reply:
Not breaking